translations.yml

```diff
@@ -1090,33 +1097,41 @@
 
 - en: 'Editors:'   # gender-neutral, plural. Precedes list of more than one person who edited the resource. If gender-neutral not in your language, use appropriate alternative, such as plural or 'Edition'.
   fr: 'Rédaction :'
+  ja: '編集者'
 
 - en: 'Update Editor:'   # gender-neutral, singular. Precedes list of one person who updates the resource. If gender-neutral not in your language, use appropriate alternative, such as plural or 'Update Edition'.
   fr: 'Rédaction des mises à jour:'
+  ja: '更新作業者'
```

Comment:
@hiroya-uga Same question as https://github.com/w3c/wai-website-data/pull/124/files#r1486346116 about the use of colons

Reply:
@remibetin I fixed it: https://github.com/w3c/wai-website-data/pull/124#discussion_r1486945810